python/main.py

```diff
@@ -1,17 +1,19 @@
 import os
 import logging
 import pathlib
-from fastapi import FastAPI, Form, HTTPException, Depends
+from fastapi import FastAPI, Form, HTTPException, Depends, UploadFile, File
 from fastapi.responses import FileResponse
 from fastapi.middleware.cors import CORSMiddleware
 import sqlite3
 from pydantic import BaseModel
 from contextlib import asynccontextmanager
-
+import json
+import hashlib
 
 # Define the path to the images & sqlite3 database
 images = pathlib.Path(__file__).parent.resolve() / "images"
 db = pathlib.Path(__file__).parent.resolve() / "db" / "mercari.sqlite3"
+items_file = pathlib.Path(__file__).parent.resolve() / "items.json"
```

Comment:
Maybe we can keep the file in `db` folder as well?
```suggestion
items_file = pathlib.Path(__file__).parent.resolve() / "db" / "items.json"
```